webrtc.html

```diff
@@ -3216,6 +3235,34 @@ <h2>Methods</h2>
                 descriptions and ICE candidate generation. This is by design for
                 legacy reasons.</p>
               </dd>
+              <dt><dfn data-idl><code>restartIce</code></dfn></dt>
+              <dd>
+                <p>The <code>restartIce</code> method tells the
+                <code><a>RTCPeerConnection</a></code> that ICE should be
+                restarted. Subsequent calls to <code>createOffer</code> will
+                create descriptions that will restart ICE, as described in
+                section 9.1.1.1 of [[!ICE]].</p>
+                <p>When this method is invoked, the user agent MUST run the
+                following steps:</p>
+                <ol>
+                  <li>
+                    <p>Let <var>connection</var> be the
+                    <code><a>RTCPeerConnection</a></code> on which the method
+                    was invoked.</p>
+                  </li>
+                  <li>
+                    <p>Empty <var>connection</var>.<a>[[\LocalUfragsToReplace]]</a>,
+                    and populate it with all ufrags found in
+                    <var>connection</var>.<a>[[\CurrentLocalDescription]]</a>,
+                    as well as all ufrags found in
+                    <var>connection</var>.<a>[[\PendingLocalDescription]]</a>.</p>
```

Comment:
Populating and comparing ufrags is not explicit here (e.g. is it a list of strings or a map of content section names and ufrags, etc) but I think this makes sense to an implementer.

Reply:
In Firefox we use a Set. We're populating here, not comparing. LMK if you want a change.

Reply:
It's also initialized to an "empty set" in this PR.